cdc/log.hh

```diff
@@ -126,6 +126,7 @@ enum class operation : int8_t {
     // enum decl, so don't change the constant values (or the datatype).
```

Comment:
(Commenting here, because I can't attach this comment to a line that is not a part of the diff)

Line 74:

```c++
// Callback to be invoked on mutation finish to fix
// the whole bit about post-image.
// TODO: decide on what the parameters are to be for this.
using result_callback = std::function<future<>()>;
```

What about this? Are we still going to use it?

Reply:
Probably not, but will remove in different change set.

Reply:
We may want to keep it for CDC metrics. @piodul please comment on this.

Reply:
I'm not sure yet. I initially thought we could use it to update the count of successful log row writes. Now, I don't think it could be used for accurate tracking of that metric, especially if multiple-mutation batches are involved. After `mutate_internal` finishes, we only are informed if everything succeeded or not (either it threw an exception, or not).

Reply:
In that case, @elcallio please don't remove it. @piodul please see if it's useful for you and if not then remove it.